pkgs/applications/misc/bottles/default.nix

```diff
@@ -55,8 +55,9 @@ python3Packages.buildPythonApplication rec {
     gobject-introspection
     gsettings-desktop-schemas
     gspell
-    gtk3
-    gtksourceview4
+    gtk4
+    gtksourceview5
+    libadwaita
     libhandy
```

Comment:
Haven't read any upstream changes but I actually don't think gtk4 apps should depend on libhandy :thinking:

Reply:
No idea either, but will try to build without it.

Reply:
It builds and start without it.

Reply:
Rolling back since it is required per upstream https://github.com/bottlesdevs/Bottles/search?q=libhandy

Reply:
> Rolling back since it is required per upstream https://github.com/bottlesdevs/Bottles/search?q=libhandy

I don't think that is required, there is no trace of libhandy in the flatpack: https://github.com/flathub/com.usebottles.bottles/blob/master/com.usebottles.bottles.yml

Reply:
I will suggest those files are out-of-date.